types/analog-clock/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for analog-clock 1.0
+// Project: https://github.com/matthewp/analog-clock#readme
+// Definitions by: ihatecsv <https://github.com/ihatecsv>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Clock class that handles the logic of the analog clock component
+export class Clock {
```

Comment:
IMO clock is not exported:
https://github.com/matthewp/analog-clock/blob/master/clock.js